notes/fintech-incrementalism-and-responsible-innovation.md

```diff
@@ -2,31 +2,45 @@
 title: Fintech Incrementalism and Responsible Innovation
 created: 2022-04-04
 date: 2022-04-04
-description: "In episode #5 of our ongoing deep dive into web3 and crypto, Rufus Pollock and Stephen Diehl explore the claim that blockchain can be a vehicle for increase in financialization through the development of more complex, blockchain-based financial products."
+description: "The fintech incrementalist position is that fintech (financial technology) is a force for effecting change in financial services and building a more stable, efficient and transparent economy. Here we evaluate the claim that blockchain-based financial technology can be a vehicle for more efficient markets through the development of more complex, blockchain-based financial products."
 image: /img/fintech-incrementalism-thumbnail.jpg
 youtube: https://www.youtube.com/watch?v=7rLQoTtwRSU&t=4s
 podcast: https://anchor.fm/life-itself/episodes/Fintech-Incrementalism-and-Responsible-Innovation-e1gn02u
 featured: true
 aliases: notes/fintech-incrementalism-and-responsible-innovation.md
 ---
 
-{/* https://www.youtube.com/watch?v=7rLQoTtwRSU&t=4s */}
+# Summary
 
-{/* Podcast: https://anchor.fm/life-itself/episodes/Fintech-Incrementalism-and-Responsible-Innovation-e1gn02u */}
+## Claim Steel-Manned
 
-***
+Building more complex financial models and products allows us to create more antifragile structures that remove or disperse [risk](../concepts/risk.md) from the broader [market](../concepts/market.md) that the public benefits from. We can see real world examples of this: mortgage-backed securities have enabled more democratic and more liquid access to mortgage credit; volaitility in the price of products has been reduced by creating future products, eg the volatility of chicken nuggets is solved by creating synthetic futures on corn and soymeal commodities that would hedge the chicken producer’s exposure to underlying price fluctuations.
 
-# Episode Notes
+Crypto is just the evolution of this trend of good incremental fintech improvemments. We want to continue moving the economy into a hyperfinanicalized 24/7 real-time always-trading market with even more complexity and lower friction than what we have presently. Blockchain can get us there. 
 
-## Summary
+Blockchain can be a vehicle for increase in financialization through the development of more complex, [blockchain](../concepts/blockchain.md)-based [financial products](../concepts/financial-asset.md). This ranges from better payment rails i.e. a better Visa, Stripe etc, to more efficient clearing systems, to full-scale innovation in financial engineering.
```

Comment:
this paragraph now seems weird in contrast to two previous. Also is this the claim evaluation for better payment rails?